lib/form_data.js

```diff
@@ -211,18 +212,19 @@ FormData.prototype._getContentDisposition = function(value, options) {
 
   var contentDisposition;
 
-  // custom filename takes precedence
+  // custom filepath or filename take precedence
   // fs- and request- streams have path property
   // formidable and the browser add a name property.
-  var filename = options.filename || value.name || value.path;
+  var filename = options.filepath || options.filename || value.name || value.path;
```

Comment:
What do you think doing it in "one-stroke" fashion?

Something like:
```
var filename = options.filepath ? path.normalize(options.filepath) : path.basename(options.filename || value.name || value.path);
```

And add `basename` call to the streams.

The main reason is to handle options here and don't "leak" them further into the function.